packages/deploy/deploy/400_asset/400_deploy_asset_auth_validator.ts

```diff
@@ -0,0 +1,17 @@
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {DeployFunction} from 'hardhat-deploy/types';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const {deployments, getNamedAccounts} = hre;
+  const {deploy} = deployments;
+
+  const {deployer, assetAdmin, backendAuthWallet} = await getNamedAccounts();
+  await deploy('AssetAuthValidator', {
+    from: deployer,
+    contract: 'AuthValidator',
```

Comment:
We have two contracts with exactly same name, how do we know we are getting the right one?

Reply:
At the moment, packages/deploy only has asset and giveaway as dependencies (not core) - does this answer your question?

Reply:
Gotcha, yup, thanks!